src/js/index.js

```diff
@@ -24,6 +24,22 @@ import ApplicationUtil from './utils/ApplicationUtil';
 
 let domElement = document.getElementById('application');
 
+// TODO: Implement loader that can concat many sprites into a single one
+setTimeout(function () {
+  var ajax = new XMLHttpRequest();
+  ajax.open('GET', 'sprite.svg', true);
+  ajax.send();
+  ajax.onload = function () {
+    var div = document.createElement('div');
+    div.innerHTML = ajax.responseText;
+    div.style.height = 0;
+    div.style.overflow = 'hidden';
+    div.style.width = 0;
+    div.style.visibility = 'hidden';
+    document.body.insertBefore(div, document.body.childNodes[0]);
+  };
+});
```

Comment:
Without this we load the sprite svg inside the `index.html` and it takes about 400ms to parse (parsing this sprite is expensive for the browser) and it's blocking as it's inline. So deferring this to load after the browser parses the JS is much preferred. It also means that it'll probably load after the CSS is received/parsed.